flexmeasures/api/common/schemas/sensor_data.py

```diff
@@ -344,11 +345,23 @@ def load_bdf(sensor_data: dict) -> BeliefsDataFrame:
         source = get_or_create_source(current_user)
         num_values = len(sensor_data["values"])
         event_resolution = sensor_data["duration"] / num_values
-        dt_index = pd.date_range(
-            sensor_data["start"],
-            periods=num_values,
-            freq=event_resolution,
-        )
+
+        if event_resolution == timedelta(hours=0):
+            if num_values > 1:
+                raise NotImplementedError(
```

Comment:
Now that I understand the case you are addressing in this PR, I think this is not a case that is just lacking an implementation on our side, but rather a case of a nonsensical time series description. Therefore (but perhaps even in either case), it should be caught during schema validation. Possibly, in `check_resolution_compatibility_of_sensor_data`, thereby slightly expanding its scope (and thus its docstring). There, we should raise a `ValidationError` for this case, which will also change the status code.